test/user.test.js

```diff
@@ -230,4 +230,42 @@ describe('USER TEST SUITE', () => {
           });
       });
   });
+  describe('Get a single User', () => {
+    it('Should get a user with valid user id present in database', (done) => {
+      chai.request(server)
+        .get('/api/v1/user/1')
+        .end((err, res) => {
+          expect(res.status).to.equal(200);
+          expect(res.body.message).to.equal('success');
+          done();
+        });
+    });
+    it('Should should not get a user when the id is not found in the database',
```

Comment:
Please, proofread the description for each of your specs.

Reply:
Ok. Will do that right away

Reply:
Thanks for that. I noticed and rectified the double appearance of `should` in about three test cases.